privates/admin.py

```diff
@@ -60,13 +64,16 @@ def get_urls(self):
 
         extra = []
         for field in self.get_private_media_fields():
-            view = self.get_private_media_view(field)
-            extra.append(
-                re_path(
-                    r"^(?P<pk>\d+)/%s/$" % field,
-                    self.admin_site.admin_view(view),
-                    name=self._get_private_media_view_name(field),
+            if field in self.no_download_fields:
+                continue
+            else:
+                view = self.get_private_media_view(field)
+                extra.append(
+                    re_path(
+                        r"^(?P<pk>\d+)/%s/$" % field,
+                        self.admin_site.admin_view(view),
+                        name=self._get_private_media_view_name(field),
+                    )
                 )
```

Comment:
please apply the "never nester" philosophy - the `continue` causes the loop iteration to terminate, so you can drop the `else` and unnest all the code inside.